src/service/bill_service.rs

```diff
@@ -227,11 +230,17 @@ impl BillService {
         identity_store: Arc<dyn IdentityStoreApi>,
         file_upload_store: Arc<dyn FileUploadStoreApi>,
     ) -> Self {
+        let network_kind = match &USERNETWORK {
```

Comment:
Hm, why do you match here? :thinking: Isn't `USERNETWORK` already a `bitcoin::Network`? I think you could just use `USERNETWORK` as `network_kind`.

Reply:
this is because of type mismatch, because USERNETWORK is return as String while we need it to return a type of Network,

Reply:
But the type of `USERNETWORK` is `bitcoin::Network` according to `main.rs`

```rust
pub static ref USERNETWORK: Network = ...
```

Reply:
Yes, the build says otherwise

when i ran cargo build, am getting a type error.

Reply:
this is crazy, using this 

`let (private_key, public_key) = util::create_bitcoin_keypair(*USERNETWORK);`
works

Reply:
I think this approach would be the best:

In `main.rs`

```rust
#[macro_use]
extern crate lazy_static;
lazy_static! {
    pub static ref CONFIG: Config = Config::try_parse().expect("Unable to fetch config");
}

```

And then, where you need a network, you use `CONFIG.bitcoin_network()`

```rust
use crate::CONFIG;

...
        let (private_key, public_key) = util::create_bitcoin_keypair(CONFIG.bitcoin_network());
```

I just checked out your branch and tested this approach and it works.